server/monitor.go

```diff
@@ -1123,8 +1123,10 @@ func (s *Server) HandleStacksz(w http.ResponseWriter, r *http.Request) {
 }
 
 type monitorIPQueue struct {
-	Pending    int `json:"pending"`
-	InProgress int `json:"in_progress,omitempty"`
+	Pending        int    `json:"pending"`
+	InProgress     int    `json:"in_progress,omitempty"`
+	Size           uint64 `json:"size,omitempty"`
```

Comment:
Reporting the `Size` is also a good idea. Can we break this down into smaller PRs so we can handle & review them as smaller units?

Reply:
Then the 3rd PR would be to add the monitoring aspect of it. Correct?